docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md

```diff
@@ -7,16 +7,27 @@ createdAt: "2020-06-03T16:02:44.317Z"
 updatedAt: "2022-12-13T20:17:44.705Z"
 ---
 
-An Edition App represents a bundle of settings, policies, backend and frontend apps that are encapsulated into a single app. These apps are created and exported by a [Sponsor Account](https://developers.vtex.com/docs/guides/vtex-io-documentation-sponsor-account), with the main objective of streamlining the installation of multiple apps essential to quickly set up a specific group of accounts.
+An Edition App represents a bundle of settings, policies, backend and frontend apps that are encapsulated into a single app. These apps are created and exported by a [Sponsor Account](https://developers.vtex.com/docs/guides/vtex-io-documentation-sponsor-account), with the main objective of streamlining the installation of multiple apps essential to set up a specific group of accounts quickly.
 
 ![EditionApp](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/vtex-io-documentation-edition-app.png)
 
 ## List of native Edition Apps
 
 The `vtex` account currently provides the following native Edition Apps:
 
-- Edition Store (`vtex.edition-store@5.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://developers.vtex.com/docs/guides/getting-started-3).
 - Edition Business (`vtex.edition-business@0.x`) - installs all the necessary apps to build a store with the [Legacy CMS Portal](https://help.vtex.com/en/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/6OCY6S9tqBXPD5mgpbBInC).
+- Edition Store (`vtex.edition-store@5.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://developers.vtex.com/docs/guides/getting-started-3).
+
+There are currently four available versions of the Edition Store. Next, you can better understand the differences between them:
+
+| Versions | What is new? |
+| --- | --- |
+|  2.x  | Standard version  |
+|  3.x  | `vtex.store-sitemap@2.x` app declared in the manifest’s [dependencies](https://developers.vtex.com/docs/guides/vtex-io-documentation-dependencies)  |
+|  4.x  | Majors of `vtex.checkout` app, including both 1.x and 2.x versions, declared in the manifest’s dependencies  |
+|  5.x  | [Search apps](https://developers.vtex.com/docs/apps/vtex.search) installed  |
```

Comment:
Acho que seria interessante listar os apps e marcar com  ❌  ou ✅ . Tipo assim:

|Version| vtex.store-sitemap | vtex.checkout | [Intelligent Search apps](https://developers.vtex.com/docs/apps/vtex.search)|
|-----|----------|--------|-----------|
|`2.x`| ❌ | ❌ | ❌ |
|`3.x`| ✅ | ❌   | ❌ |
|`4.x`| ❌ | ✅  |  ❌ |
|`3.x`| ✅ | ✅  | ✅ |

Reply:
Fica mais visual e claro o que cada versão tem/não tem

Reply:
Fiz o ajuste, ficou assim:

There are currently four available versions of the Edition Store. Next, you can better understand their differences regarding the apps installed by default in the manifest’s [dependencies](https://developers.vtex.com/docs/guides/vtex-io-documentation-dependencies).

| Version	| [vtex.store-sitemap](https://developers.vtex.com/docs/apps/vtex.store-sitemap)	| vtex.checkout (including 1.x and 2.x  versions) | [vtex.search](https://developers.vtex.com/docs/apps/vtex.search)
|   :---:   |   :---:   |   :---:   |   :---:   |
| 2.x	| ❌ | ❌	| ❌ |
| 3.x	| ✅	| ❌	| ❌ |
| 4.x	| ✅	| ✅	| ❌ |
| 5.x	| ✅    | ✅	| ✅ |